### PR TITLE
refine dm docs

### PR DESCRIPTION
**Pull request by IANTHEREAL**
### What problem does this PR solve? 
adjust some dm document as feedbacks

### Check List 

Tests 
 - No code

### CODE REVIEW

**Comment by csuzhangxc** [comment]
LGTM

File: docs/dm/zh_CN/configuration/configuration.md [hunk 1]
```diff
@@ -64,14 +67,23 @@ filters:                                            # 上游数据库实例的
     ​    table-pattern: "t_*"
     ​    events: ["truncate table", "drop table"]
     ​    action: Ignore
+    user-filter-2:
```
Comment (amyangfei) [inline docs/dm/zh_CN/configuration/configuration.md]: s/user-filter-2/filter-rule-2
it is better to keep same name regulation with `filter-rule-1` ?

File: docs/dm/zh_CN/restrictions.md [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
 ===
 
-### 数据库版本
+### 任务配置检查
 
-- 5.5 < MySQL version < 5.8
-- MariaDB version >= 10.1.2
-
-另外，在使用 dmctl 启动任务时，DM 会对任务上下游数据库的配置、权限等进行前置检查，具体细节见参考 [同步任务前置检查](./precheck.md)
+在使用 dmctl 启动任务时，DM 会对任务上下游数据库的配置、权限等进行前置检查，具体细节见参考 [同步任务前置检查](./precheck.md)
```
Comment (amyangfei) [inline docs/dm/zh_CN/restrictions.md]:
```suggestion
在使用 dmctl 启动任务时，DM 会对任务上下游数据库的配置、权限等进行前置检查，具体细节参考 [同步任务前置检查](./precheck.md)
```

**Review by amyangfei — COMMENTED** [review]
LGTM

**Pull request merged**